public/js/controllers/app.controller.js

```diff
@@ -1,6 +1,6 @@
-import { UIController } from "./UI.js";
-import { DataController } from "./data.js";
-import { AuthController } from "./auth.js";
+import { UIController } from "./UI.controller.js";
```

Comment:
```suggestion
import { UIController } from "./ui_controller";
```
(The '.' is a bit confusing here because . usually separates the name from the type in file names, so better to use '_')

Reply:
Updated thanks